addOns/retire/src/test/java/org/zaproxy/addon/retire/UpstreamJsRepositoryUnitTest.java

```diff
@@ -31,7 +38,29 @@ class UpstreamJsRepositoryUnitTest {
     void shouldParseUpstreamJsRepository() {
         // Given
         String path = "/org/zaproxy/addon/retire/resources/jsrepository.json";
-        // When / Then
-        assertDoesNotThrow(() -> new Repo(path));
+        Reader data;
+        Map<String, RepoEntry> entries = null;
+        // When
+        try {
+            data = RetireTestUtils.reader(path);
+            entries = Repo.createEntries(data);
+        } catch (IOException e) {
+            fail();
+        }
+        // Then
+        entries.values()
```

Comment:
I'd suggest using enhanced for statements to avoid the code span which is not that readable.